models/action.go

```diff
@@ -136,6 +137,16 @@ func (a Action) GetIssueInfos() []string {
 	return strings.SplitN(a.Content, "|", 2)
 }
 
+func (a Action) GetIssueTitle() string {
+	issueID := com.StrTo(a.GetIssueInfos()[0]).MustInt64()
```

Comment:
But I know how to fix it, you can leave it to me or I can indicate where changes should be made. Which way you prefer?